logstash-core/src/main/java/org/logstash/Valuefier.java

```diff
@@ -44,16 +42,16 @@ private static Object convertJavaProxy(JavaProxy jp) {
         }
     }
 
-    public static Object convertNonCollection(Object o) {
+    private static Object convertNonCollection(Object o) {
         try {
-            return o == null ? BiValues.NULL_BI_VALUE : BiValues.newBiValue(o);
+            return BiValues.newBiValue(o);
```

Comment:
I only added this upstream check to improve JIT behavior (with some visible success), but after recent changes, it became more visible that we can save some of these checks like this one.